_includes/fancyboxtemplate.html

```diff
@@ -1,4 +1,5 @@
-{{ include.gallery }}
+Gallery: {{ include.gallery }}
+Thumbnail URL: {{ site.galleries[include.gallery].images[0].thumbnail }}
 {% for image in site.galleries[include.gallery].images %}
```

Comment:
I was actually worried that Jekyll wasn't passing parent variables to the included template *at all*.